.ci/scripts/disp_s1/test_int_disp_s1.sh

```diff
@@ -129,7 +129,11 @@ do
         overall_status=1
     else
         # Retrieve the return code written to disk by the comparison script
-        overall_status=$(cat "$output_dir/compare_disp_s1_products.rc")
+        test_status=$(cat "$output_dir/compare_dswx_hls_products.rc")
+
+        if [ $test_status -ne 0 ]; then
```

Comment:
Just out of curiosity, can we accomplish this with a logical assignment operator? Something like `overall_status = $overall_status | $test_status`?